README.md

```diff
@@ -2,6 +2,18 @@
 
 [![license](https://img.shields.io/badge/license-Apache2-green.svg?style=flat)](https://raw.githubusercontent.com/mfucci/node-matter/master/LICENSE) 
 
+## THIS REPOSITORY HAS BEEN MOVED!!
```

Comment:
@JimBuzbee @vves @javierbarellano4v @jdesai61 @Hedda @lboue @dberardo @andyearnshaw @lboue @Flavien FYI, as contributors to code and discussions please feel informed about what will happen here soon ... But in fact we just join forces with project chip to make it even better :-)

Reply:
So node-matter will be a project under packages/  in matter.js?

Reply:
Yes, and we will slowly move down to packages/matter.js all the code which is not node.js specific.

Reply:
@jdesai61 Yes, I just merged it. it will be in packages/matter-node.js and also the new package when we first publish will be names @project-chip/matter-node.js